packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -22,6 +22,11 @@ export interface CarouselState {
     activeId: string;
 }
 
+enum SlideTransitionDirection {
+    previous = "previous",
+    next = "next",
+}
```

Comment:
doesnt the FlipperDirection enum have the same values as this enum?

Reply:
Sure, I can use that